pyproject.toml

```
@@ -442,13 +443,14 @@ commands =
     clean: make clean
     make debug
 
-[testenv:py{39,310,311}-docs-live]
+[testenv:py{39,310,311,312}-docs-live]
 # tip: remove apidocs before using this feature (`cd docs; make clean`)
 description = Build the documentation and launch browser (with live updates)
 deps =
     py39: -rrequirements/requirements-py-3.9.txt
     py310: -rrequirements/requirements-py-3.10.txt
     py311: -rrequirements/requirements-py-3.11.txt
+    py312: -rrequirements/requirements-py-3.11.txt
```

```suggestion
    py312: -rrequirements/requirements-py-3.12.txt
```

@sphuber, no, there are two (three in fact, "type" -> "typo") :smiley: